common/lighthouse_macros/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "lighthouse_macros"
```

Comment:
This crate name is a bit generic and may encourage adding random macros into this crate.
What about something more specific, like `workspace_meta` or `workspace_crates_info`

Reply:
I actually named it that with the idea that if we needed other Lighthouse-specific/misc macros, they could reuse the crate instead of creating a new one :thinking: 
Happy to change it of course

Reply:
Ended up changing the name anyway :sweat_smile: